test/extended/openstack/loadbalancers.go

```diff
@@ -674,6 +719,9 @@ func createTestDeployment(depName string, labels map[string]string, replicas int
 	isFalse := false
 	isTrue := true
 	netExecParams := "--" + strings.ToLower(string(protocol)) + "-port=" + fmt.Sprintf("%d", port)
+	if protocol == v1.ProtocolTCP {
```

Comment:
nit: to improve readability, you could treat all cases at the same time (e.g. with a case/switch) instead of setting a default and then override for TCP.

Something like:
```Go
	var netExecParams string
	switch protocol {
	case v1.ProtocolTCP:
		netExecParams = fmt.Sprintf("--http-port=%d", port)
	default:
		netExecParams = fmt.Sprintf("--%s-port=%d", strings.ToLower(string(protocol)), port)
	}
```

Reply:
<3 it. Thank you.